src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -173,19 +140,174 @@ public void toggleOrientation() {
     isFieldOriented = !isFieldOriented;
     if (!isFieldOriented) {
       // robot oriented drive means we can't hold a field oriented heading
-      endAlignment();
     }
   }
 
-  public void setAlignment(Rotation2d rotation) {
-    alignmentRotation = rotation;
+  public Command applyRequest(Supplier<SwerveRequest> requestSupplier) {
+    return run(() -> this.setControl(requestSupplier.get())).withName("Request Supplier");
   }
 
-  public void endAlignment() {
-    alignmentRotation = null;
+  /**
+   * Request the robot to point in a specified direction any non zero rotation demand will result in
```

Comment:
This is confusing and should be split into two sentences:

```suggestion
   * Request the robot to point in a specified direction. Any non zero rotation demand will result in
```